File: specification/specs/x_pack/security/api_key/invalidate_api_key/InvalidateApiKeyRequest.ts
@rest_spec_name("security.invalidate_api_key")
class InvalidateApiKeyRequest extends RequestBase {
  query_parameters?: {
  }
  body?: {
    id?: string;
    name?: string;
    owner?: boolean;
    realm_name?: string;
    username?: string;
  }
}

File: output/csharp/XPack/Security/ApiKey/InvalidateApiKey/InvalidateApiKeyRequest.cs
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class InvalidateApiKeyRequest : RequestBase {
		
		[DataMember(Name="id")]
		public string Id { get; set; }


		[DataMember(Name="name")]
		public string Name { get; set; }


		[DataMember(Name="owner")]
		public bool Owner { get; set; }


		[DataMember(Name="realm_name")]
		public string RealmName { get; set; }


		[DataMember(Name="username")]
		public string Username { get; set; }

	}
}
